common/account_utils/src/validator_definitions.rs

```diff
@@ -361,6 +396,30 @@ mod tests {
     use super::*;
     use std::str::FromStr;
 
+    #[test]
+    fn validator_definitions_file_locks() {
+        let validators_dir: PathBuf = ".".into();
+
+        /* create validator definitions file */
+        let blank_validators: ValidatorDefinitions = ValidatorDefinitions::create(&validators_dir)
+            .expect("Failed to create empty validator definitions type");
+        blank_validators
+            .save(&validators_dir)
+            .expect("Failed to save validator definitions file");
+
+        let _some_validators: ValidatorDefinitions = ValidatorDefinitions::open(&validators_dir)
+            .expect("Failed to open validator definitions file");
```

Comment:
With the fixes applied to `::create` this step will fail, because `blank_validators` still holds the lock. I think we can just delete this step